Resources/doc/mailchimp.md

```diff
@@ -9,15 +9,15 @@ First of all you need to install drewn's Mailchimp API, if this bundle is missin
 ```json
 {
     "require": {
-        "drewm/mailchimp-api": "^2.2"
+        "drewm/mailchimp-api": "^2.5"
```

Comment:
why this BC Break?

Reply:
2.5 is the newest version. Just to bring it up to date